src/solvers/thermal_operators.hpp

```diff
@@ -100,14 +94,14 @@ class DynamicConductionOperator : public mfem::TimeDependentOperator {
   AlgebraicSolver T_solver_;
 
   /**
-   * @brief Pointer to the assembled M matrix
+   * @brief Non-owning pointer to the assembled M matrix
```

Comment:
This is a good convention, so that intellisense can inform people about ownership on autocompletion without needing to dive into documentation.